kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-drools/src/main/java/org/kie/server/remote/rest/drools/CommandResource.java

```diff
@@ -88,20 +89,33 @@ public Response manageContainer(@Context HttpHeaders headers,
             format = MarshallingFormat.valueOf(contentType);
         }
         logger.debug("Received request with content '{}'", cmdPayload);
-        Object result = delegate.callContainer(id, cmdPayload, format, classType);
         Header conversationIdHeader = buildConversationIdHeader(id, registry, headers);
+        @SuppressWarnings("squid:S3740")
+        ServiceResponse<?> result = delegate.callContainer(id, cmdPayload, format, classType);
+        Status status;
+        switch (result.getType()) {
+            case FAILURE:
+                status = Response.Status.INTERNAL_SERVER_ERROR;
+                break;
+            case NO_RESPONSE:
```

Comment:
Can't see where this `NO_RESPONSE` code is thrown from the `callContainer` call as per

https://github.com/kiegroup/droolsjbpm-integration/blob/master/kie-server-parent/kie-server-services/kie-server-services-drools/src/main/java/org/kie/server/services/drools/DroolsKieContainerCommandServiceImpl.java#L46-L80 it seems it only throws either `SUCCESS` or `FAILURE` codes.

Am I missing something?

Reply:
yes, me neither, but since NO RESPONSE is a valid member of the enumeration, I feel the code should be handling it just in case.
The other option was to ignore it and consider the request valid, returning ok like in the sucess case, but I feel it was cheating
wdyt?

Reply:
ok, I see. Makes sense. Thanks